src/endpoints/applecrashreport.rs

```diff
@@ -117,7 +117,7 @@ fn handle_apple_crash_report_request(
         let response_future = request_future
             .and_then(clone!(symbolication, |mut request| {
                 if request.sources.is_none() {
-                    request.sources = Some((*default_sources).clone());
+                    request.sources = Some(default_sources.to_vec());
```

Comment:
before we had a clone here, but I think the `to_vec` re-allocates as well?
Is it possible to use a borrow for `request.sources`? no idea what type that has.

Reply:
Before this was cloning the underlying vector, so actually copying the data.  Now it's also copying this into a new vector.  I actually think this became much more readable.

Reply:
It is exactly the same. Cloning a vector or calling `to_vec` on a slice that is backed by a vector is identical.